ecr/pusher_test.go

```diff
@@ -64,31 +64,47 @@ func TestPushManifestReturnsManifestWriter(t *testing.T) {
 	} {
 		t.Run(mediaType, func(t *testing.T) {
 			callCount := 0
+			desc := ocispec.Descriptor{
```

Comment:
Can you move this back down to where it was?  That keeps the test data close to the test callsite (this is the argument to `pusher.Push`).

Reply:
Yeah I'll update this. It was being used for consistencies' sake [here](https://github.com/awslabs/amazon-ecr-containerd-resolver/blob/53a30030c3d5ccf097cb965f5946f2365b949413/ecr/pusher_test.go#L89) but that can simply be swapped out with `mediaType` from the loop.